CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+## v1.2.0 (2021 Nov 8)
+
+* Support for Rails 6.1
+* Bumped ruby patch version
```

Comment:
I think the `.ruby-version` might only be for development purposes and wouldn't affect a consumer of the gem. We _probably_ wouldn't need to mention it in the changelog?